files/build_templates/sonic_debian_extension.j2

```diff
@@ -317,8 +317,7 @@ sudo LANG=C chroot $FILESYSTEM_ROOT umount -lf /sys
 {% endif %}
 
 ## Install Systemd generator for SONiC services
-sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/systemd-sonic-generator*.deb || \
-    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/systemd-sonic-generator*.deb
 
```

Comment:
both remove

Reply:
If I remove both then the generator doesn't get installed in the image

Reply:
that is why I'd like to change to use INSTALL instead of DEPENDS